### ISSUE TITLE
Malformed response with repeated fields

### ISSUE THREAD

**Issue by antonioorct** [tatethurston/TwirpScript#109]
Below is an example .proto file when this occurs:
```protobuf
syntax = "proto3";
package test;

message Bar {
  int32 id = 1;
  string name = 2;
}

service FooService {
  rpc GetBar(GetBarRequest) returns (GetBarResponse);
}

message GetBarRequest {}
message GetBarResponse {
  repeated Bar bars = 1;
}
```

After making the request on the client i receive the correct length of the array as formed on the server but each element of the array is undefined.

Server side: 
```ts
return { bars: [{ id: 0, name: "First", id: 1, name: "Second"}] };
```
Client side:
```ts
// log output
[undefined, undefined]
```

EDIT: forgot to mention, i'm using v0.0.42

**Comment by arranf** [comment]
@antonioorct Which version(s) of TwirpScript are you using?

**Comment by antonioorct** [comment]
v0.0.42

**Comment by arranf** [comment]
@tatethurston I can replicate this!

**Comment by tatethurston** [comment]
@antonioorct Awesome, thanks for reporting this. And thanks @arranf for digging in -- I'll take a look as well.

**Comment by tatethurston** [comment]
Ah 🤦 I see the issue. I'll have a patch submitted shortly.

**Comment by tatethurston** [comment]
This is fixed in v0.0.43, LMK if you encounter any issues.
